### PR TITLE
Feature/texrenaming

### PR DESCRIPTION
**Pull request by rpspringuel**
As I was getting ready to start Phase III, I found a few User level command which I had missed in Phase II.

This PR fixes those.  Since these are functions I'm less familiar with (that's why I forgot about them), please check my documentation carefully to make sure I've got accurate descriptions.

I'm currently having testing problems, so I'm not sure that this is entirely bug free (I've already caught a couple).  We should probably hold off on merging until the testing problems are ironed out so that we can check for more bugs.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -100,6 +100,17 @@ As of v3.0.0 this project adheres to [Semantic Versioning](http://semver.org/).
 - `\GreUseZeroHyphen`, supplanted by `\greseteolhyphen{zero}`
 - `\greremoveclef`, supplanted by `\gresetclef{invisible}`
 - `\grenormalclef`, supplanted by `\gresetclef{visible}`
+- `\AddHEpisemusBridges`, supplanted by `\gresethepisemus{bridge}`
+- `\RemoveHEpisemusBridges`, supplanted by `\gresethepisemus{break}`
+- `\UseAlternatePuncutmCavum`, supplanted by `\gresetpunctumcavum{alternate}`
```
Comment (henryso) [inline CHANGELOG.md]: Punctum is misspelled.

File: tex/gregoriotex-signs.tex [hunk 2]
```diff
@@ -926,23 +926,32 @@
...
-\relax %
+  \gre@deprecated{\protect\AddHEpisemusBridges}{\protect\gresethepisemus{bridge}}%
+  \gre@hepisemusbridgestrue%
 }%
 
 \def\RemoveHEpisemusBridges{%
-\xdef\greaddhepisemusbridges{0}%
-\relax %
+  \gre@deprecated{\protect\RemoveHepisemusBridges}{\protect\gresethepisemus{break}}
 }%
```
Comment (eschwab) [inline tex/gregoriotex-signs.tex]: Need to add `\gre@hepisemusbridgefalse` here?

File: tex/gregoriotex-signs.tex [hunk 3]
```diff
@@ -926,23 +926,32 @@
...
 
+\newif\ifgre@hepisemusbridge
+\gre@hepisemusbridgetrue
+
 \def\AddHEpisemusBridges{%
-\xdef\greaddhepisemusbridges{1}%
-\relax %
+  \gre@deprecated{\protect\AddHEpisemusBridges}{\protect\gresethepisemus{bridge}}%
+  \gre@hepisemusbridgestrue%
```
Comment (eschwab) [inline tex/gregoriotex-signs.tex]: The bool is singular: `\gre@hepisemusbrigdetrue`

File: tex/gregoriotex-chars.tex [hunk 4]
```diff
@@ -182,73 +182,100 @@
...
+      \greresetglyph{CustosDownLong}%
+      \greresetglyph{CustosDownShort}%
+      \greresetglyph{CustosDownMedium}%
+      \greresetglyph{BarBrace}%
+      \greresetglyph{Punctum}%
+      \relax %
+    }%
+    {\IfStrEq{#1}{medicaea}%
+      {%
+        \gresetglyphstyle{defualt}\greusestylecommon %
```
Comment (henryso) [inline tex/gregoriotex-chars.tex]: defualt should be default

File: tex/gregoriotex-chars.tex [hunk 5]
```diff
@@ -182,73 +182,100 @@
...
-\greresetglyph{CustosUpShort}%
-\greresetglyph{CustosUpMedium}%
-\greresetglyph{CustosDownLong}%
-\greresetglyph{CustosDownShort}%
-\greresetglyph{CustosDownMedium}%
-\greresetglyph{BarBrace}%
-\greresetglyph{Punctum}%
-\relax %
+  \gre@deprecated{\protect\greusedefaultstyle}{\protect\gresetglyphstyle{default}}%
+  \gresetglyphstyle{default}
```
Comment (henryso) [inline tex/gregoriotex-chars.tex]: missing % at the end; similar for the others that follow in this file

**Comment by henryso** [comment]
I can reproduce the problem you mentioned in https://github.com/gregorio-project/gregorio-test/issues/3 with four of the tests using this branch (but not using develop).  I'll see if I can track it down.

File: tex/gregoriotex-signs.tex [hunk 6]
```diff
@@ -926,23 +926,33 @@
...
-\xdef\greaddhepisemusbridges{1}%
-\relax %
+  \gre@deprecated{\protect\AddHEpisemusBridges}{\protect\gresethepisemus{bridge}}%
+  \gre@hepisemusbridgetrue%
 }%
 
 \def\RemoveHEpisemusBridges{%
-\xdef\greaddhepisemusbridges{0}%
-\relax %
+  \gre@deprecated{\protect\RemoveHepisemusBridges}{\protect\gresethepisemus{break}}
```
Comment (henryso) [inline tex/gregoriotex-signs.tex]: Missing % at the end of the line

File: tex/gregoriotex-signs.tex [hunk 7]
```diff
@@ -1471,20 +1481,34 @@
...
+}%	
+
+
 \def\UseAlternatePunctumCavum{%
-\grechangeglyph{PunctumCavum}{greciliae}{.caeciliae}%
-\grechangeglyph{LineaPunctumCavum}{greciliae}{.caeciliae}%
-\grechangeglyph{PunctumCavumHole}{greciliae}{.caeciliae}%
-\grechangeglyph{LineaPunctumCavumHole}{greciliae}{.caeciliae}%
-\relax %
+  \gre@deprecated{\protect\UseAlternatePunctumCavum}{\protect\gresetpuncutmcavum{alternate}} %
```
Comment (henryso) [inline tex/gregoriotex-signs.tex]: Spurious space before the %

File: tex/gregoriotex-signs.tex [hunk 8]
```diff
@@ -1471,20 +1481,34 @@
...
+  \gresetpunctumcavum{alternate}%
 }%
 
 \def\UseNormalPunctumCavum{%
-\greresetglyph{PunctumCavum}%
-\greresetglyph{LineaPunctumCavum}%
-\greresetglyph{PunctumCavumHole}%
-\greresetglyph{LineaPunctumCavumHole}%
-\relax %
+  \gre@deprecated{\protect\UseNormalPunctumCavum}{\protect\gresetpuncutmcavum{normal}} %
```
Comment (henryso) [inline tex/gregoriotex-signs.tex]: Spurious space before the %

**Comment by henryso** [comment]
The unable to open font error went away when I removed the `var` directory under `gregorio-test`.  In order to prevent luaotfload from needing to re-cache stuff every run, I modified `gregorio-test.sh` (a few weeks back) to use `rsync` instead of `cp` when installing GregorioTeX in the local directory (`var/texmf` under `gregorio-test`).  However, when switching branches, file dates can confuse rsync and the files are then mismatched.  Try removing the `var` directory to see if that fixes your testing problems.

**Comment by henryso** [comment]
With the changes I mentioned in the diffs applied, all the tests pass.

**Comment by rpspringuel** [comment]
I'm getting some test failures, but they all appear to be related to changing expectations regarding the braces, so that appears to be a different issue.

The one exception:
`tex-output/bugs/fix-396` gives me a differs from expected failure, but when I open the `diff-page.png` there doesn't appear to be anything highlighted:

diff-page

Given the lack of highlighted differences in the above image, I have no idea why this test is failing.

**Comment by henryso** [comment]
There are differences, but they are miniscule due to very slight changes in the fonts.  Generally when I see things like this, I consider it acceptable and accept the result as new expectation.  I'm working on updated tests for the braces, but I won't be able to get to them until this evening.

**Comment by rpspringuel** [comment]
Okay, then I'd say this is ready for merge.

**Comment by eschwab** [comment]
Let's do it then. Looks good to me.

**Comment by henryso** [comment]
FYI: The differences in https://github.com/gregorio-project/gregorio/pull/531#issuecomment-118661634 are in the italic letters ("T.P." and "Ps.").  I'm guessing there is some slight differences in the Libertine fonts.  I was using Fr. Peron's test directly, but maybe I should switch it to use the "locked-down" Alegreya fonts in the gregorio-test project.

**Pull request merged**